tests/unittests/unittests.cc

```diff
@@ -85,31 +84,30 @@ class CheckHeader1Test : public ::testing::Test {
     }
 
     KTX_header testHeader;
+
   public:
     static ktx_uint8_t ktxId[12];
 };
 
-ktx_uint8_t CheckHeader1Test::ktxId[12] = {
-    0xAB, 0x4B, 0x54, 0x58, 0x20, 0x31, 0x31, 0xBB, 0x0D, 0x0A, 0x1A, 0x0A
-};
+ktx_uint8_t CheckHeader1Test::ktxId[12] = {0xAB, 0x4B, 0x54, 0x58, 0x20, 0x31,
```

Comment:
Is there a way to fix the format of a few lines in a file? This represents the signature of a KTX v1 file so it much easier for humans to follow as originally written.

Reply:
You can exclude lines of code from being formatted by surrounding them with:

```cpp
// clang-format off
...
// clang-format on
```

There are examples of this in the generated files.